File: python/06-funcoes/main097.py
''' Refaça o exercício 91, só que agora em forma de função Maior(), mas faça uma 
adaptação que vai receber TRÊS números como parâmetro e vai retornar qual foi o 
maior entre eles.

Data 11/11/2025 -> 49ª Dia
'''

def Maior(num1,num2,num3):
    maior = 0
    if num1 > num2 and num1 > num3:
        maior = num1
    elif num2 > num3 and num2 > num1:
        maior = num2
    elif num3 > num1 and num3 > num2:
        maior = num3
    return maior
num1 = int(input('Digite o primeiro numero: '))
num2 = int(input('Digite o segundo numero: '))
num3 = int(input('Digite o terceiro numero: '))
print(Maior(num1,num2,num3))

File: c/06-funcoes/main097.c
/*Refaça o exercício 91, só que agora em forma de função Maior(), mas faça uma 
adaptação que vai receber TRÊS números como parâmetro e vai retornar qual foi o 
maior entre eles.

Data 11/11/2025 -> 49ª Dia
*/

#include <stdio.h>

int Maior(int num1, int num2, int num3){

    int maior = 0;

    if(num1 > num2 && num1 > num3){

        maior = num1;
    }
    else if(num2 > num3 && num2 > num1){

        maior = num2;
    }
    else if(num3 > num1 && num3 > num2){

        maior = num3;
    }
    return maior;
}
int main(){

    int num1,num2,num3;

    printf("Digite o primeiro numero: ");
    scanf("%d",&num1);

    printf("Digite o segundo numero: ");
    scanf("%d",&num2);

    printf("Digite o terceiro numero: ");
    scanf("%d",&num3);

    int resultado = Maior(num1,num2,num3);

    printf("%d",resultado);
    return 0;
}